src/main/jbake/content/blog/2016/11/16/hawkular-apm-opentracing-js.adoc

```diff
@@ -0,0 +1,73 @@
+= Hawkular APM OpenTracing JavaScript
+Pavol Loffay
+2016-11-16
+:icons: font
+:jbake-type: post
+:jbake-status: published
+:jbake-tags: blog, APM, microservices, distributed tracing, javascript
+
+In the latest https://github.com/hawkular/hawkular-apm/releases/tag/0.12.0.Final[release]
+of Hawkular APM we have added a new OpenTracing implementation. This
+time we decided to add JavaScript provider, therefore, now it is possible to get timing metrics
+from a polyglot environment instrumented with our Java and JavaScript OpenTracing implementations.
+
+== Library
+Library is implemented against the latest (0.13.0) version of OpenTracing JavaScript API.
+Data is reported to APM via HTTP. For debugging it is possible to use a console recorder.
+From the sampling perspective it allows developers to add their own sampling mechanism.
+In the future releases we will add more built-in samplers.
+
+The following code snippet shows how to initialize and create spans in JavaScript code. It works for
+a web browser and Node.js application.
+
+``` javascript
```

Comment:
Adoc uses ---- to surround code (and yes I see that it is working like you did it) and you can use  

```
.Title of the snipped
[source, javascript]
----
my code
---- 

```